athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/handlers/GlueMetadataHandler.java

```diff
@@ -330,12 +333,18 @@ protected GetTableResponse doGetTable(BlockAllocator blockAllocator, GetTableReq
         }
 
         SchemaBuilder schemaBuilder = SchemaBuilder.newBuilder();
+        Set<String> excludeKey = new HashSet<>();
+        excludeKey.add(DATETIME_FORMAT_MAPPING_PROPERTY);
         if (table.getParameters() != null) {
-            table.getParameters().entrySet().forEach(next -> schemaBuilder.addMetadata(next.getKey(), next.getValue()));
+            table.getParameters()
+                    .entrySet()
+                    .forEach(next -> schemaBuilder.addMetadata(next.getKey(), next.getValue(), excludeKey));
         }
 
         //A column name mapping can be provided to get around restrictive Glue naming rules
         Map<String, String> columnNameMapping = getColumnNameMapping(table);
+        Map<String, String> dateTimeFormatMapping = getDateTimeFormatMapping(table);
+        Map<String, String> datetimeFormatMappingWithColumnName = new HashMap<>();
```

Comment:
instead of multiple maps, maybe we can create a column object that holds the info we care about?

Reply:
Im not sure what you mean by creating a column object. dateTimeFormatMapping holds the raw map from the table, and datetimeFormatMappingWithColumnName is something GlueMetadataHandler builds based on dateTimeFormatMapping with "normalized" column name